app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :Likes, foreign_key: 'posts_id'
+  has_many :comments, foreign_key: 'posts_id'
+
+  belongs_to :author, class_name: 'User', foreign_key: 'users_id'
+
+  after_save :increment_author_posts_couter
+
+  def increment_author_posts_couter
+    author.increment!(:posts_count)
+  end
```

Comment:
- [ ] Since you are calling this method in `after_save` calback`, it would be very nice if you make it private then